testFlight1/lib/teensyShared/thresholds.h

```diff
@@ -1,18 +1,25 @@
 #ifndef THRESHOLDS_H
 #define THRESHOLDS_H
-
+ 
 //TODO set values for the thresholds
-float coast_time_thresh = 500; // .5 seconds, probably maybe
-float coast_thresh = -9; //"-9" because below -9.81 must be freefall, add some buffer for data noise
+float coast_time_thresh = 300;
+float coast_thresh = 0;
 float apogee_thresh = 0;
+float main_thresh_min = -2; //still not determined
+float main_thresh_max = -1; //still not determined
+float drogue_thresh = -2; //still not determined
+float landed_thresh = -.1;
 float coast_timer;
-float apogee_timer;
 float descent_timer;
 float burn_timer; //Measuring how long the burn happens
-float launch_az_thresh = 3500; //Minimum vert acc. for launch detection
+float launch_az_thresh = 1; //Minimum vert acc. for launch detection
 float launch_time_thresh = 300; //Minimum time to confirm launch
-float apogee_time_thresh = 500;
+float apogee_time_thresh = 300;
+float drogue_time_thresh = 2000;
+float main_time_thresh = 2000;
```

Comment:
Make this 3 seconds, the main coasts for a long time so again, 3 seconds is not bad. This also gives it enough time to not accidentally enter this stage before drogue, because if that happens we'll miss drogue altogether.